windows/src/global/delphi/cust/CustomisationMessages.pas

```diff
@@ -187,7 +187,7 @@ procedure TCustomisationMessageManager.LoadLocale;
   FDefaultLocaleDoc.async := False;
   FDefaultLocaleDoc.validateOnParse := False;
 
-  FPath := GetDebugPath('xmltemplate '+ExtractFileName(FCustStorageFileName), ExtractFilePath(FCustStorageFileName));
+  FPath := GetDebugPath('KeymanConfigStaticHttpFilesPath');
```

Comment:
This was a bug, because we now use this constant `KeymanConfigStaticHttpFilesPath` for debug paths for the http static files elsewhere.